utils/formatErrorMessage.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Produce a cleanly-formatted error message from a caught error:
+ * removes ANSI, collapses white space, and truncates at 100 chars.
+ *
+ * @param {*} err - An exception object, or anything else caught by try/catch.
+ * @param {boolean} [asciiOnly=true] - Whether to restrict output to the ASCII.
+ * @returns {string} A well-behaved message.
+ */
+export function formatErrorMessage (err, asciiOnly = true) {
+  let message
+
+  if (err instanceof Error) {
+    message = err.message
+  } else if (err instanceof String) {
+    message = err
+  } else {
+    message = String(err)
+  }
+
+  // strip ansi
+  // https://stackoverflow.com/a/29497680
+  const ansiRegex = /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g
```

Comment:
@rebeccacremona Please don't hesitate to add `//eslint-disable-line` so Standard JS stops barking at this 😄 .

Reply:
Ah! Thanks for the tip!